lib/metadata/util/win32/Win32Software.rb

```diff
@@ -179,7 +180,7 @@ def registry_patches_packages(hotfix, element)
       return if element.attributes.nil? || element.attributes[:keyname].nil?
       if element.attributes[:keyname][0, 8] == 'Package_'
         # don't add this package if the ID is nil
-        return if (hotfix_id = hotfix_id(e)).nil?
```

Comment:
@hsong-rh Same here, was this refactored or something that caused the breakage?

Reply:
Same change as above: https://github.com/ManageIQ/manageiq-smartstate/pull/71